Handymen now check if a tile is being serviced by another handyman before starting their task (#26639)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Change: [#26689] A clear scenario editor can now be opened directly from the command line.
- Fix: [#26494] The limits for where walls on the path side of the first tile block the view of rides/scenery are wrong (original bug).
- Fix: [#26504] The boosts/penalties to excitement and nausea that air time provides are calculated wrong.

> - Fix: [#26639] Handymen now check if a tile is already being serviced by another handyman before starting their task.

Reviewer: For fixes, please describe the problem instead of the new behaviour. Something like:

```suggestion
- Fix: [#26639] Handymen could begin a task while another handyman was already doing the exact same task.
```

- Improved: [#26765] The ride operations tab now visually separates fields using group boxes.
- Fix: [#26756] Guests cannot puke or litter on sloped path.
- Fix: [#26758] [Plugin] IPv6 addresses are reported incorrectly.

> - Fix: [#26639] Handymen could begin a task while another handyman was already doing the exact same task.

Reviewer: The CI keeps failing, because the order of the changelog is not right. This should fix it:

```suggestion
- Fix: [#26639] Handymen could begin a task while another handyman was already doing the exact same task.
- Fix: [#26756] Guests cannot puke or litter on sloped path.
- Fix: [#26758] [Plugin] IPv6 addresses are reported incorrectly.
```
